src/electron-forge-package.js

```diff
@@ -65,6 +66,10 @@ const main = async () => {
         prepareSpinner = ora.ora(`Preparing to Package Application for arch: ${(prepareCounter === 2 ? 'armv7l' : 'x64').cyan}`).start();
       }
       await fs.remove(path.resolve(buildPath, 'node_modules/electron-compile/test'));
+      const bins = await pify(glob)(path.join(buildPath, '**/.bin/**/*'));
+      for (const bin of bins) {
+        await fs.remove(bin);
+      }
```

Comment:
Hmmm. Good point. For some reason I thought that Electron Packager ran `npm install` but it was just for its tests.